test/scripts/init-mongo.js

```diff
@@ -10,3 +10,12 @@ db.createUser({
         },
     ],
 });
+
+db.createUser({
+    user: "mongodb_exporter",
+    pwd: "s3cr3tpassw0rd",
+    roles: [
+        { role: "clusterMonitor", db: "admin" },
+        { role: "read", db: "local" }
+    ]
+})
```

Comment:
```suggestion
})

```

Newline at end of file